cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,80 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
-
+const { faker } = require('@faker-js/faker');
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+const homePage = new HomePageObject(); 
+const settingsPage = new SettingsPageObject();
+const signInPage = new SignInPageObject();
+let user;
 describe('Settings page', () => {
   before(() => {
-
-  });
-
+    cy.task('db:clear');
+  })
+  
   beforeEach(() => {
-
-  });
+    homePage.visitMainPage;
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    }); 
+    
+  })
 
   it('should provide an ability to update username', () => {
-
-  });
+    let username = faker.person.firstName();
+    homePage.loginUser(user.email, user.username, user.password);
+    homePage.visitMainPage;
+    homePage.clickSettingsLink;
+    settingsPage.writeNewUsername(username);
```

Comment:
```suggestion
    homePage.loginUser(user.email, user.username, user.password);
    homePage.visitMainPage;
    homePage.clickSettingsLink;
    settingsPage.writeNewUsername(username);
```
you could move it to beforeEach hook